markdown/ch5.markdown

```diff
@@ -1175,14 +1175,14 @@ For example, if every GT segment $g$ is split into two equally-sized
 segments $a_1$ and $a_2$ in AS, then $H(AS|GT) = 1$, because after knowing a
 pixel is in $g$, you
 still need 1 additional bit to know whether it belongs to $a_1$ or $a_2$.
-However, H(GT|AS) = 0, because regardless of whether a pixel is in $a_1$ or
+However, $H(GT | AS) = 0$, because regardless of whether a pixel is in $a_1$ or
```

Comment:
I recently read a paper that had this as "S" (segmentation) and "T" (truth). Nice, consecutive, single letters. =) Can you update this to that notation?